www/blog/2019/05/13/geonames/index.md

```diff
@@ -0,0 +1,223 @@
+  - `gn:admin4_code` = admin4_code
+  - `gn:population` = population
+  - `gn:elevation` = elevation
+  - `gn:dem` = dem
+  - `gn:timezone` = timezone
+  - `gn:modification_date` = modification_date
+```
+
+These properties (along with the GeoNames-sourced properties) gave us the building blocks to then export this GeoJSON file using the handy [exportify tool](https://github.com/whosonfirst/py-mapzen-whosonfirst-export/blob/master/scripts/wof-exportify) to create and validate a Who's On First record with the minimum viable properties.
+
+Since Who's On First bakes the idea of a place's hierarchy into a record's property, we used a point-in-polygon (PIP) service (some notes about that [here](https://github.com/whosonfirst/whosonfirst-cookbook/blob/master/how_to/setting_up_pip_service.md)) to update each new record's hierarchy.
+
+In the case of Ahinco, Chile, this allows Who's On First to store a full administrative hierarchy for this locality (see below).
+
+```
+    "wof:hierarchy":[
+        {
+            "continent_id":102191577, (South America)
+            "country_id":85633057, (Chile)
+            "region_id":85682233, (De Los Lagos)
+            "county_id":102063135, (Llanquihue)
+            "locality_id":1226313821 (Ahinco)
+        }
+    ],
+```
+
+### Other examples
+
+This work is not limited to Chile, like the Ahinco example above. Other examples of new locality records include [Abeikeng](https://spelunker.whosonfirst.org/id/1225865875/), a locality in [Fujian](https://spelunker.whosonfirst.org/id/85669735/), [China](https://spelunker.whosonfirst.org/id/85632695/) and 
+[Zela](https://spelunker.whosonfirst.org/id/1360321381/), a locality in [Metekel](https://spelunker.whosonfirst.org/id/1108695191/), [Ethiopia](https://spelunker.whosonfirst.org/id/85632257/).
+
+In fact, prior to the GeoNames work, Who's On First maintained zero records at the locality placetype within Metekel, Ethiopia. This work added forty seven new locality records - like [Debre Zeyit](https://spelunker.whosonfirst.org/id/1210052931/)
+ and [Dibate](https://spelunker.whosonfirst.org/id/1277019105/). Here are the new locality descendants of Metekel, Ethiopia, visualized:
+
+![](https://user-images.githubusercontent.com/18567700/51141067-01732980-17fd-11e9-9b9d-286132692d67.png?raw=true)
+
+## Next steps
+
+While this import of new locality records added a tremendous amount of new data to Who's On First, there is additional work to be completed. In some cases, GeoNames has a property flag that has been used inconsistently or a record has been misclassified as a locality when in Who's On First it should actually be stored at the neighbourhood placetype.
+
+Additionally, the `gn:` properties in many of these records will be scrubbed to ensure Who's On First correctly stores source values. Eventually, the concordance values in any new locality record will be used to add more information about each place as possible (adding more names from Wikipedia, for example).
```

Comment:
You mean for future GN updates for existing properties, or new properties?

Reply:
What does "scrub" mean here?

Reply:
Replaced and clarified the first sentence in recent commit.